### DISCUSSION TITLE
speed increase idea

### DISCUSSION THREAD

**Discussion by joran520** [Saskia-Oosterbroek/decona#6] [Ideas]
I have read multiple articles about the creation/use of consesus sequences.
These studied the effects of read depth on the quality/accuracy of the concensus sequences.
What they found wast that after a 100-300 sequences the overal accuracy of the read did not improve much further.
In some cases it decreased as more systematic error got the overhand.
It would be nice to have an additional parameter for a possible subset of each cluster.
As together with the cluster size it could reduce the computational time for large datasets which produce lots of clusters of a large size.

**Comment by Saskia-Oosterbroek** [comment]
Hi Joran,

This is a good idea! I have been thinking about taking a subset but I didn't want it to be hard coded. 
It should also be properly randomized as I believe the first part of your sequencing run favours shorter reads somehow. 
Definitely something to think about!

**Reply by AlexGaithuma** [reply]
Additionally, you could use diamond instead of blast for taxonomy. It's much faster. 
Also think of making it possible to stop the pipeline and continue from last instance.
